docs/coordinates/skycoord.rst

```diff
@@ -324,23 +324,23 @@ documentation::
   >>> sc = SkyCoord(1, 2, frame='icrs', unit='deg', obstime='2013-01-02 14:25:36')
   >>> sc.<TAB>  # doctest: +SKIP
   sc.T                                   sc.match_to_catalog_3d
```

Comment:
It looks like you didn't change the name of `frame_attributes` in `BaseCoordinateFrame` (which I think is fine - keep it that way) but then I don't understand this change. `frame_attributes` should still be the name on SkyCoord...